src/parser.ts

```diff
@@ -266,10 +266,9 @@ export class Parser {
     if (exp.flags & ts.SymbolFlags.Alias) {
       targetSymbol = this.checker.getAliasedSymbol(exp);
     }
-    const declaration =
-      targetSymbol.valueDeclaration || targetSymbol.declarations![0];
```

Comment:
`targetSymbol.declarations` can be undefined. `valueDeclaration` should contain everything I need if it exists.

Reply:
Tests passed, so hopefully this works for others as well.